Helper/GraphQL/Constants.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+
+namespace Bolt\Boltpay\Helper\GraphQL;
+
+
+class Constants {
```

Comment:
RequestConstants? I prefer name to be specific so this class won't be dump all

Reply:
its already within GQL namespace right? i want to put all GQL constants here